pkg/output/output.go

```diff
@@ -216,3 +221,22 @@ func uniqueNormalizeCertNames(names []string) []string {
 	}
 	return results
 }
+
+// mistmatchedCert returns if cert names does not contain host
+func mistmatchedCert(host string, names []string) bool {
+	builder := &bytes.Buffer{}
+	for _, name := range names {
+		builder.WriteString("^")
+		builder.WriteString(strings.ReplaceAll(name, "*.", "[^. ]*."))
```

Comment:
instead of building regexes dynamically, we should consider better approaches for comparison as this seems quite expensive and redundant